docs/HPCCSystemAdmin/SA-Mods/SecMgrMod.xml

```diff
@@ -31,7 +31,7 @@
 { 
     ISecManager * createInstance(const char *serviceName,
                                  IPropertyTree &amp;secMgrCfg,
-                                 IPropertyTree &amp;authCfg); 
```

Comment:
@rpastrana This looks fine. However if you make the change @RussWhitehead  suggested to use bind instead of bnd, it needs to be made here, as well.